mistralrs-core/src/xlora_models/gemma.rs

```diff
@@ -561,16 +561,12 @@ impl XLoraModel {
         tgt_len: usize,
         seqlen_offset: usize,
```

Comment:
This function already passed the `seqlen_offset`. I think it was already being correctly calculated for the current use-case, but I'm not sure